tracker/views.py

```diff
@@ -43,3 +45,36 @@ def post(self, request, *args, **kwargs) -> HttpResponse:
         [messages.error(request, error_) for error_ in form.errors.values()]
 
         return render(request, self.template_name, {"form": form})
+
+
+class ContributorListView(View):
```

Comment:
Use `ListView` it's more appropriate class and add `get_queryset` and `render_to_response`

Reply:
addressed in [5a753c0](https://github.com/KostCode-Organization/hackathon_bot/pull/44/commits/5a753c0f83ae3af67c925a787f325f6f36598714)